### PR TITLE
Slack Notification for Live Testnet Test Runs

### PR DESCRIPTION
**Pull request by kalverra**

### CODE REVIEW

File: .github/workflows/integration-tests.yml [hunk 1]
```diff
@@ -432,6 +434,15 @@ jobs:
...
           QA_KUBECONFIG: ${{ secrets.QA_KUBECONFIG }}
+      - name: Notify Slack
+        if: always()
+        env:
+          SLACK_CHANNEL: ${{ secrets.QA_SLACK_CHANNEL }}
+          SLACK_USERNAME: Chainlink Live Release Tester
+          SLACK_COLOR: ${{ job.status }}
+          SLACK_TITLE: Test
+          SLACK_MESSAGE: "Result for OCR Smoke Test on ${{ matrix.testnet }} is ${{ job.status }}"
+        uses: rtCamp/action-slack-notify@12e36fc
```
Comment (AnieeG) [inline .github/workflows/integration-tests.yml:445]: Did you already check `slackapi/slack-github-action`?

File: .github/workflows/integration-tests.yml [hunk 2]
```diff
@@ -432,6 +432,48 @@ jobs:
           QA_AWS_REGION: ${{ secrets.QA_AWS_REGION }}
           QA_AWS_ROLE_TO_ASSUME: ${{ secrets.QA_AWS_ROLE_TO_ASSUME }}
           QA_KUBECONFIG: ${{ secrets.QA_KUBECONFIG }}
+      - name: Notify Slack
+        if: always()
+        id: slack
+        uses: slackapi/slack-github-action@007b2c3c751a190b6f0f040e47ed024deaa72844
```
Comment (chainchad) [inline .github/workflows/integration-tests.yml:438]:
```suggestion
        uses: slackapi/slack-github-action@007b2c3c751a190b6f0f040e47ed024deaa72844 # v1.23.0
```

Adding the version tag like this also means that dependabot can bump the version annotation as well.

**Review by chainchad — APPROVED** [review]
Small nit about actions version comment. Looks good though, thanks for doing this.

File: .github/workflows/integration-tests.yml (hunk 2 above)
Comment (kalverra) [reply .github/workflows/integration-tests.yml]: Added, ty

**Review by chainchad — APPROVED** [review]

**Pull request merged**